packages/connectivity_plus_linux/pubspec.yaml

```diff
@@ -2,24 +2,25 @@ name: connectivity_plus_linux
 description: Linux implementation of the connectivity_plus plugin
 homepage: https://plus.fluttercommunity.dev/
 repository: https://github.com/fluttercommunity/plus_plugins/tree/main/packages/
-version: 0.3.1
+version: 0.4.0-nullsafety.0
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
```

Comment:
Should I change it to this version also in other `connectivity_plus` packages?

Reply:
Let's try that and see if the CI passes

Reply:
Now it's saying 
```
ERR : The current Dart SDK version is 2.12.0-259.12.beta.
    | 
    | Because connectivity_plus requires SDK version >=2.12.0-259.14.beta <3.0.0, version solving failed.

Weird 🤷‍♂️😃
```

Reply:
Looks like CI is running on an older Dart version?

```
ERR : The current Dart SDK version is 2.12.0-259.12.beta.
    | 
    | Because connectivity_plus requires SDK version >=2.12.0-259.14.beta <3.0.0, version solving failed.
```